docs/utils/getCssVariablesStartingWith.ts

```diff
@@ -3,7 +3,7 @@ export function getCssVariablesStartingWith(prefix: string) {
 
   // Iterate through all style sheets in the document
   for (const styleSheet of Array.from(document.styleSheets).filter(
-    (s) => !s.href
+    (s) => !s.href || s.href.startsWith(window.location.origin)
```

Comment:
Interesting, was it trying to get cross-origin stylesheets?

Reply:
Yeah, exactly, i added this checking because of error message, which we catch in case getting stylesheets from another origin, but it excluded our stylesheets too, so i was need to avoid excluding our origin, adding `startsWith` function check. :)
Do you have any concerns on this?

Reply:
My only thought here is that it might be better to parse the href instead of using `startsWith` since I'm not sure how compatible this might be cross browser. Something like:

```js
new URL(s.href).origin === window.location.origin
```

It might need to be wrapped in a try/catch since `new URL` can thrown.